### DISCUSSION TITLE
How do I get my ScrollArea to work?

### DISCUSSION THREAD

**Discussion by shirakaba** [nodegui/svelte-nodegui#46] [Q&A]
Copying this Spectrum post from @johanalkstal:

> I can't get the scrolling to activate on my ScrollArea and I don't know why.
> If I resize the window so that the content doesn't fit, the scroll bar doesn't activate.
> Gist: https://gist.github.com/johanalkstal/7c0d2fd29d712e95bfea0e75fd85b200

**Comment by shirakaba** [comment]
I've just realised this is more of a NodeGUI question than a Svelte NodeGUI one. We can discuss it here until NodeGUI sets up GitHub Discussions, I suppose.

**Reply by johanalkstal** [reply]
Thanks for picking it up!
